pattern_library/views.py

```diff
@@ -2,6 +2,7 @@
 from django.template.loader import get_template
 from django.utils.html import escape
```

Comment:
Just need to add the following import here:
`from django.utils.decorators import method_decorator`